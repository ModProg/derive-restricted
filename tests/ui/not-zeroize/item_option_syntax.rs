use std::marker::PhantomData;

use derive_where::derive_where;

#[derive_where = "invalid"]
struct WrongAttributeSyntax<T>(PhantomData<T>);

fn main() {}
