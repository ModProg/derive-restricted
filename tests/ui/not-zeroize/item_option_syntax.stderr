error: key-value macro attributes are not supported
 --> tests/ui/not-zeroize/item_option_syntax.rs:5:1
  |
5 | #[derive_where = "invalid"]
  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^

error: unsupported trait syntax, expected one of Clone, Copy, Debug, Default, Eq, Hash, Ord, PartialEq, PartialOrd
 --> tests/ui/not-zeroize/item_option_syntax.rs:5:18
  |
5 | #[derive_where = "invalid"]
  |                  ^^^^^^^^^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/ui/not-zeroize/item_option_syntax.rs:6:48
  |
6 | struct WrongAttributeSyntax<T>(PhantomData<T>);
  |                                                ^ consider adding a `main` function to `$DIR/tests/ui/not-zeroize/item_option_syntax.rs`
